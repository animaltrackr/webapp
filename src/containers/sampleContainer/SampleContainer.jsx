import React, { Component } from 'react';
import PropTypes from 'prop-types';

class SampleContainer extends Component {
	render() {
		return <div>SampleContainer</div>;
	}
}

SampleContainer.propTypes = {};

export default SampleContainer;
