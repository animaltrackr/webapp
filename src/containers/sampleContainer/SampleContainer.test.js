import React from 'react';
import { shallow } from 'enzyme';
import SampleContainer from './SampleContainer';

describe('<SampleContainer />', () => {
	test('renders', () => {
		const wrapper = shallow(<SampleContainer />);
		expect(wrapper).toMatchSnapshot();
	});
});
