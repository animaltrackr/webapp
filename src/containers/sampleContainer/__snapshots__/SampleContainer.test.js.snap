// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SampleContainer /> renders 1`] = `ShallowWrapper {}`;
